lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -110,15 +110,18 @@ defmodule Membrane.MP4.Muxer.CMAF do
 
   @impl true
   def handle_end_of_stream(:input, ctx, state) do
-    last_timestamp = hd(state.samples).metadata.timestamp
-    {buffer, state} = generate_segment(ctx.pads.input.caps, %{timestamp: last_timestamp}, state)
+    last_dts = hd(state.samples).dts
+
+    {buffer, state} =
+      generate_segment(ctx.pads.input.caps, %{dts: last_dts, payload: <<>>}, state)
```

Comment:
Why are you passing here `%{dts: last_dts, payload: <<>>}` instead of just `%{dts: last_dts}`?

Reply:
Right, in fact it's sufficient to pass just `last_dts` to this function